target/linux/ath79/dts/ar9330_dlink_dir-505.dts

```diff
@@ -108,12 +108,32 @@
 				label = "art";
 				reg = <0x10000 0x10000>;
 				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					cal_art_1000: calibration@1000 {
+						reg = <0x1000 0x440>;
+					};
+				};
 			};
 
-			mac: partition@20000 {
+			partition@20000 {
 				label = "mac";
 				reg = <0x20000 0x10000>;
 				read-only;
+
+				nvmem-layout {
+					compatible = "fixed-layout";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					macaddr_mac_4: calibration@4 {
+						reg = <0x4 0x11>;
```

Comment:
Use `"mac-base"`, the same reason. Please grep `0x11>;` to check them all.

Reply:
Fixed.